[2025-10-04T23:28:32.975+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_data_dag.ingest_orders_data manual__2025-10-04T23:17:44.635799+00:00 [queued]>
[2025-10-04T23:28:32.983+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_data_dag.ingest_orders_data manual__2025-10-04T23:17:44.635799+00:00 [queued]>
[2025-10-04T23:28:32.984+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 3
[2025-10-04T23:28:32.997+0000] {taskinstance.py:2191} INFO - Executing <Task(S3ToSqlOperator): ingest_orders_data> on 2025-10-04 23:17:44.635799+00:00
[2025-10-04T23:28:33.003+0000] {standard_task_runner.py:60} INFO - Started process 2462 to run task
[2025-10-04T23:28:33.007+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ingest_data_dag', 'ingest_orders_data', 'manual__2025-10-04T23:17:44.635799+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/dag_ingest_data.py', '--cfg-path', '/tmp/tmp7d6bybd8']
[2025-10-04T23:28:33.010+0000] {standard_task_runner.py:88} INFO - Job 19: Subtask ingest_orders_data
[2025-10-04T23:28:33.070+0000] {task_command.py:423} INFO - Running <TaskInstance: ingest_data_dag.ingest_orders_data manual__2025-10-04T23:17:44.635799+00:00 [running]> on host ff034d7c1964
[2025-10-04T23:28:33.163+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingest_data_dag' AIRFLOW_CTX_TASK_ID='ingest_orders_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-04T23:17:44.635799+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-04T23:17:44.635799+00:00'
[2025-10-04T23:28:33.165+0000] {s3_to_sql.py:105} INFO - Loading orders.csv to SQL table "staging"."orders"...
[2025-10-04T23:28:33.204+0000] {base.py:83} INFO - Using connection ID 'minio_s3' for task execution.
[2025-10-04T23:28:33.205+0000] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-10-04T23:28:33.490+0000] {base.py:83} INFO - Using connection ID 'postgres_warehouse' for task execution.
[2025-10-04T23:28:33.509+0000] {base.py:83} INFO - Using connection ID 'postgres_warehouse' for task execution.
[2025-10-04T23:28:33.657+0000] {sql.py:544} INFO - Loaded 1000 rows into "staging"."orders" so far
[2025-10-04T23:28:33.812+0000] {sql.py:544} INFO - Loaded 2000 rows into "staging"."orders" so far
[2025-10-04T23:28:33.958+0000] {sql.py:544} INFO - Loaded 3000 rows into "staging"."orders" so far
[2025-10-04T23:28:34.121+0000] {sql.py:544} INFO - Loaded 4000 rows into "staging"."orders" so far
[2025-10-04T23:28:34.380+0000] {sql.py:544} INFO - Loaded 5000 rows into "staging"."orders" so far
[2025-10-04T23:28:34.380+0000] {sql.py:547} INFO - Done loading. Loaded a total of 5000 rows into "staging"."orders"
[2025-10-04T23:28:34.391+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ingest_data_dag, task_id=ingest_orders_data, execution_date=20251004T231744, start_date=20251004T232832, end_date=20251004T232834
[2025-10-04T23:28:34.428+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-04T23:28:34.444+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
